ui/components/MeshModelRegistry/MeshModelDetails.js

```diff
@@ -157,10 +161,30 @@ const ModelContents = withSuppressedErrorBoundary(({ modelDef }) => {
   const orderdMetadataRight = reorderObjectProperties(metaDataRight, orderRight);
   const isShowStatusSelector = !Array.isArray(modelDef?.model.version);
 
+  const handleExport = () => {
+    const a = document.createElement('a');
+    a.href = '/api/meshmodels/export?id=' + modelDef.id;
+    document.body.appendChild(a);
+    a.click();
+    a.remove();
+  };
+
   return (
     <div>
-      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+      <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
         <TitleWithImg displayName={modelDef.displayName} iconSrc={modelDef?.metadata?.svgColor} />
+        {ExportAvailable ? (
+          <UsesSistent>
+            <Button
+              onClick={handleExport}
+              variant="contained"
+              startIcon={<DownloadIcon />}
```

Comment:
Icon should be white (should support user-selected color theme).

@captain-Akshay can you assist here?